apps/advisor/kinds/check.cue

```diff
@@ -0,0 +1,36 @@
+package advisor
+
+datasourcecheck: {
+	kind:	   "Check"
+	pluralName: "Checks"
+	current:	"v0alpha1"
+	versions: {
+		"v0alpha1": {
+			codegen: {
+				frontend: false
+				backend:  true
+			}
+			schema: {
+				spec: {
+					// Generic data input that a check can receive
+					data?: [string]: string
+				}
+				status: {
+					report: {
+						// Number of elements analyzed
+						count: int
+						// List of errors
+						errors: [...{
+							// Investigation or Action recommended (severity of the error)
+							type: "investigation" | "action"
+							// Human readable reason for the error
+							reason: string
+							// Action to take to resolve the error
+							action: string
+						}]
```

Comment:
Nit: Will a field named `action` not be confusing when it's also a type?

Reply:
That's a good point, I think it's better to rename `type` with `severity` and have two levels (high == needs action and low == investigation recommended). The UI can give more details about that.